### DISCUSSION TITLE
is Inventory Planning Module included in the Open Source Community Version?

### DISCUSSION THREAD

**Discussion by shawsa-artc** [frePPLe/frepple#631] [General]
I've installed the docker compose version and it seems offering lesser features compare to the demo site, for eg, i couldnt find the Inventory Planning module.

Do you have like a comparison chart on the features offering between paid and community version ?

**Comment by jdetaeye** [comment]
This comparison is available at https://frepple.com/editions/

There is also a thread earlier in this forum on this topic.

**Reply by shawsa-artc** [reply]
Thank you very much for the info !
